dotCMS/src/main/java/com/dotmarketing/common/util/SQLUtil.java

```diff
@@ -34,6 +34,11 @@
 	private static SecurityLoggerServiceAPI securityLoggerServiceAPI =
 			APILocator.getSecurityLogger();
 
+	public static final String ASC  = "asc";
+	public static final String DESC  = "desc";
+	public static final String _ASC  = " " + ASC ;
+	public static final String _DESC  = " " + DESC;

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)

Reply:
does not matter we need to say there is a space at the begin